try:
    a = int(input('Numerador: '))
    b = int(input('Denominador: '))
    r = a / b
except Exception as erro:
    print('Infelizmente tivemos um problema. O problema encontrado foi {erro.__class__}')
else:
    print(f'O resultado Ã© {r:.1f}')
finally:
    print('Volte sempre. Muito obrigada.')